src/browser/provider/built-in/chrome/config.js

```diff
@@ -1,100 +1,41 @@
 import emulatedDevices from 'chrome-emulated-devices-list';
-import OS from 'os-family';
-import { find as findElement, pickBy as filterProperties } from 'lodash';
+import { pickBy as filterProperties } from 'lodash';
+import {
+    hasMatch, findMatch, isMatchTrue, getModes, splitEscaped, getPathFromParsedModes, parseConfig
+} from '../../utils/argument-parsing';
 
 
 const HEADLESS_DEFAULT_WIDTH  = 1280;
 const HEADLESS_DEFAULT_HEIGHT = 800;
 
-const CONFIG_TERMINATOR_RE = /(\s+|^)-/;
+const MODES_LIST = ['userProfile', 'headless', 'emulation'];
```

Comment:
I feel `MODE_LIST` is more correctly grammatically.
But maybe it's more appropriate to use `AVAILABLE_MODES` here